packages/plugin/vite/src/VitePlugin.ts

```diff
@@ -67,7 +67,12 @@ export default class VitePlugin extends PluginBase<VitePluginConfig> {
     if (VitePlugin.alreadyStarted) return false;
     VitePlugin.alreadyStarted = true;
 
-    await fs.rmdir(this.baseDir, { recursive: true });
+    try {
+      // Node.js v16+ must check in advance if the directory exists, while v14 works fine
+      await fs.access(this.baseDir);
+      await fs.rmdir(this.baseDir, { recursive: true });
```

Comment:
Since we're doing a Node 16 fix:

https://nodejs.org/api/deprecations.html#dep0147-fsrmdirpath--recursive-true-

```
In future versions of Node.js, recursive option will be ignored for fs.rmdir, fs.rmdirSync, and fs.promises.rmdir.

Use fs.rm(path, { recursive: true, force: true }), fs.rmSync(path, { recursive: true, force: true }) or fs.promises.rm(path, { recursive: true, force: true }) instead.
```